tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
```

Comment:
The use of passive "al" make this seem unnatural to me.
Suggestion: "${{user}} uzis ${{name}}n kontraŭ ${{target}}!"